src/pcapi/admin/custom_views/offer_view.py

```diff
@@ -134,6 +156,70 @@ class OfferView(BaseAdminView):
     form_columns = ["criteria", "rankingWeight", "isEducational"]
     simple_list_pager = True
 
+    @action("bulk_edit", "Bulk Edit")
```

Comment:
(détail)

Je traduirais bien "bulk edit". ;)

```suggestion
    @action("bulk_edit", "Édition multiple")
```

(ou bien "Édition en masse", ou un truc dans le même genre)